homeassistant/components/sensor/systemmonitor.py

```diff
@@ -164,3 +166,5 @@ def update(self):
         elif self.type == 'since_last_boot':
             self._state = dt_util.utcnow() - dt_util.utc_from_timestamp(
                 psutil.boot_time())
+        elif self.type == 'load':
+            self._state = '{:.1f}, {:.1f}, {:.1f}'.format(*os.getloadavg())
```

Comment:
This will not play nice with the frontend if you are using a badge as it will be displayed as `12.23, 12.89, 13.09`.